app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,10 @@
+from app.players.player import Player
+
+
+class Dwarf(Player):
```

Comment:
![image](https://user-images.githubusercontent.com/103201010/219366626-fb0a38c0-20cf-4dab-b7ae-1752a9c9caaf.png)
fix this

Reply:
still not fixed